browser/net/brave_site_hacks_network_delegate_helper.cc

```diff
@@ -125,25 +105,34 @@ void ApplyPotentialQueryStringFilter(std::shared_ptr<BraveRequestInfo> ctx) {
     return;
   }
 
-  // Regular expressions are dirty and error-prone, but unfortunately
```

Comment:
I'd suggest keeping this comment, though it needs to be modified slightly because the code isn't using regular expressions anymore, as a reminder that we cannot use the built-in functions to parse HTML5-form query strings.

Assuming that query strings have a given format (or encoding) is a common (and easy-to-make) mistake.

Reply:
Good idea -- I'm keeping a new version of this comment.